packages/flutter_tools/test/commands/test_test.dart

```diff
@@ -48,6 +48,17 @@ void main() {
       return _testFile('ticker', automatedTestsDirectory, flutterTestDirectory);
     }, skip: io.Platform.isWindows); // Dart on Windows has trouble with unicode characters in output
 
+    testUsingContext('report which user created widget caused the error', () async {
+      Cache.flutterRoot = '../..';
+      return _testFile('print_user_created_ancestor', automatedTestsDirectory, flutterTestDirectory,
+        extraArgs: const <String>['--track-widget-creation']);
+    }, skip: io.Platform.isWindows); // Dart on Windows has trouble with unicode characters in output
+
+    testUsingContext('report which user created widget caused the error - no flag', () async {
+      Cache.flutterRoot = '../..';
+      return _testFile('print_user_created_ancestor_no_flag', automatedTestsDirectory, flutterTestDirectory);
+    }, skip: io.Platform.isWindows); // Dart on Windows has trouble with unicode characters in output
```

Comment:
please link to the relevant issue on github with a TODO

Reply:
I copied from other tests, I am actually not sure what is it referring to, let me remove the skip and see what will happen